src/api/queries/blockchainQueries.js

```diff
@@ -226,3 +226,19 @@ ApiClient_.prototype.fetchEthTopTransactions = function (slug, from, to, limit,
 
   return this.conn.graphQLQuery(query, 'projectBySlug')
 }
+
+ApiClient_.prototype.fetchEthSpentOverTime = function (slug, from, to, limit, transactionType) {
```

Comment:
Probably these arguments are leftovers? They are not used anywhere.

Reply:
Yes, you're right, sorry about that.

Reply:
Should be okay now.